public/res_primaryLanguage.json

```diff
@@ -11,26 +11,22 @@
     },
     "projects": [
       {
-        "title": "Animal Shelter",
-        "startDate": "2020",
-        "description": "The most expanded application I had opportunity to work with. I've learned many technologies and my code was reviewed by awesome curator. Application handles all adoption processess and allows to store all evidence on adopting animals from animal shelter.",
+        "title": "Todo App",
+        "startDate": "2021",
+        "description": "I learned a lot about SOLID principles doing this Odin bootcamp project. I love to be organized. That is why with a friend developer I build an extended todo app - with Firebase background structure, and more functionality. We also created an UML diagramm for optimizing the architecture.",
```

Comment:
"I love to be organized (GTD rocks!)."

Maybe you can already add the link to the repo of the new app even if it's not ready yet

Reply:
Nice addition with GTD =). 

We really haven't come far yet with implementing stuff... Maybe at a later point I would add a link...